packages/mds-policy-service/service/provider.ts

```diff
@@ -57,20 +62,14 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService> & ProcessCont
     serviceToPolicyRepository('writePolicyMetadata', () =>
       PolicyRepository.writePolicyMetadata(validatePolicyMetadataDomainModel(policy_metadata))
     ),
-  publishPolicy: async (policy_id, publish_date) => {
-    try {
+  publishPolicy: (policy_id, publish_date) =>
+    serviceToPolicyRepository('publishPolicy', async () => {
```

Comment:
Nit unrelated to this current line, I think that because this is `serviceToPolicyRepository` method is being used to do more than just wrap `repository` calls, we ought to pick an even more abstract name `serviceErrorWrapper` or something. Nothing about the `serviceToPolicyRepository` function itself really enforces that it's just wrapping a single repo method call

Reply:
better name! yes :)